libs/ett/auth/data-access-api/src/lib/oidc/auth-http.service.ts

```diff
@@ -25,23 +25,35 @@ import { Observable } from 'rxjs';
 import { AuthOidcQueryParameterName } from './auth-oidc-query-parameter-name';
 
```

Comment:
question:

Is it by design that we keep interfaces inside the same file as the service?

Are we using seperate files for interfaces if they i.eg. would be larger?

Reply:
It's intentional when they're only relevant to this service and its consumers. `AuthOidcQueryParameterName` is used for other concerns so it's in a separate file.